examples/pub-sub-history/page.md

```diff
@@ -0,0 +1,120 @@
+# Message history with Pub/Sub
+
+History enables users to retrieve historical messages published to an application.
+
+Use history to retrieve messages previously published to a channel within an application. Users can retrieve the history for messages such as a bidding history, or status updates.
+
+History enables users to retrieve historically published messages within an application, allowing them to understand how the current state has been achieved. For example, in an auction site the user can see the history of bids as well as a see the current active bid. Within a sports application with live statistics the user will be able to see how the game has progressed so far.
+
+History is implemented using [Ably Pub/Sub](https://ably.com/docs/products/channels). The Ably SDK contains a set of purpose-built APIs that abstract away the complexities involved in architecting Pub/Sub features.
```

Comment:
`The Ably SDK contains a set of purpose-built APIs that abstract away the complexities involved in architecting Pub/Sub features.` 

I think this sentence needs replacing for Pub/Sub as it's too abstraction-focussed towards the other products. Maybe something along the lines of:

`The Pub/Sub SDK provides a set of flexible APIs capable of building any realtime application. It is powered by Ably's reliable and scalable platform.` ?